src/include/kdbmacros.h

```diff
@@ -73,21 +82,30 @@
 			Key * c = ksNext (info);                                                                                           \
 			if (!c)                                                                                                            \
 			{                                                                                                                  \
-				ELEKTRA_SET_ERRORF (84, error, "the key %s (value %s) was added", keyName (k), keyString (k));             \
+				/* TODO: Correct? Maybe Interface? */                                                                      \
+				ELEKTRA_SET_INSTALLATION_ERRORF (                                                                          \
+					error, "Read only plugin, kdbSet not supported but the key %s (value %s) was added", keyName (k),  \
```

Comment:
but it was tried ... (because it did not succeed?)

Reply:
See above

Reply:
> but it was tried ... (because it did not succeed?)

Actually after thinking twice about the sentence I am not sure what exactly you mean. Reading through the code actually shows me that setting a value actually succeeded but shouldn't?

Reply:
not supported but the key %s (value %s) was added" -> not supported but the key %s (value %s) was tried to be added"